apps/frontend/src/app/3_organisms/EmailNotificationSettingsDialog/EmailNotificationSettingsDialog.tsx

```diff
@@ -294,64 +304,96 @@ const EmailNotificationSettingsDialogComponent: React.FC<
         title={t(translations.emailNotificationsDialog.dialogTitle)}
       />
       <DialogBody className="p-6">
-        <div className="p-6 bg-gray-90">
-          <Paragraph style={ParagraphStyle.tall}>
-            {t(translations.emailNotificationsDialog.title)}
-          </Paragraph>
-          <FormGroup
-            className="mt-6 mb-4"
-            label={t(translations.emailNotificationsDialog.emailInputLabel)}
-            errorLabel={
-              hasUnconfirmedEmail
-                ? t(
-                    translations.emailNotificationsDialog
-                      .unconfirmedEmailWarning,
-                  )
-                : undefined
-            }
-          >
-            <Input
-              value={email}
-              onChangeText={setEmail}
-              placeholder={t(
-                translations.emailNotificationsDialog.emailInputPlaceholder,
-              )}
-              disabled={loading || !notificationToken}
-              dataAttribute="alert-signup-email"
+        {hasUnconfirmedEmail ? (
```

Comment:
```suggestion
        {hasUnconfirmedEmail && isOnboarding ? (
```
In your current implementation, you would see this message also in a normal notification settings dialog.

Also, you broke the functionality that was there before (showing a warning under the email input in case email was unverified), fix it.

Reply:
I've already discussed it with soulbit we want to show it on normal flow as well